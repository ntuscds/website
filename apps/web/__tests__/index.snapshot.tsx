import { render } from '@testing-library/react'
import { ChakraProvider } from "@chakra-ui/react";
import Home from '@/pages/index'

it('renders homepage unchanged', () => {
  const { container } = render(
      <ChakraProvider>
        <Home />
      </ChakraProvider>
  )
  expect(container).toMatchSnapshot()
})
