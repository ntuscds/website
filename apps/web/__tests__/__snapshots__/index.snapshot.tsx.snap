// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="css-1u7tgf3"
  >
    <div
      class="chakra-stack css-nqtk8r"
    >
      <div
        class="chakra-stack css-1dgewg3"
      >
        <p
          class="chakra-text css-17en64"
          role="heading"
        >
          WELCOME TO SCSE CLUB
        </p>
        <div
          class="chakra-stack css-md5zo3"
        >
          <a
            class="chakra-button css-jd3l4b"
            href="/events"
          >
            <p
              class="chakra-text css-0"
            >
              LEARN MORE
            </p>
          </a>
          <a
            class="chakra-button css-jd3l4b"
            href="/contact"
          >
            <p
              class="chakra-text css-0"
            >
              CONTACT US
            </p>
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    class="chakra-stack css-12nxbtz"
  >
    <div
      class="css-1jbryaa"
    >
      <div
        class="css-0"
      >
        <div
          class="css-1vz34qo"
        >
          <a
            class="chakra-link css-13jvj27"
            href="blog/techterview-101-navigating-the-internship-world"
          >
            <div
              class="css-hkp7"
            >
              <div
                class="css-1m6ql1p"
              >
                <img
                  alt="Techterview 101: Navigating the Internship World"
                  sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
                  src="https://clubs.ntu.edu.sg/csec/techterview-101-navigating-the-internship-world/cover-photo/"
                  style="object-fit: fill;"
                />
              </div>
              <div
                class="chakra-stack css-lha1ht"
              >
                <h2
                  class="chakra-heading css-zr62dw"
                >
                  Techterview 101: Navigating the Internship World
                </h2>
                <p
                  class="chakra-text css-v04ol"
                >
                  February 22, 2022
                </p>
                <p
                  class="chakra-text css-q9k0mw"
                >
                  Techterview 101: Navigating the Internship World, a senior sharing event for the SCSE students took place on the 24th of...
                </p>
              </div>
            </div>
          </a>
        </div>
      </div>
      <div
        class="css-0"
      >
        <div
          class="css-1vz34qo"
        >
          <a
            class="chakra-link css-13jvj27"
            href="blog/21st-main-committee"
          >
            <div
              class="css-hkp7"
            >
              <div
                class="css-1m6ql1p"
              >
                <img
                  alt="21st Main Committee"
                  sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
                  src="https://clubs.ntu.edu.sg/csec/untitled/"
                  style="object-fit: fill;"
                />
              </div>
              <div
                class="chakra-stack css-lha1ht"
              >
                <h2
                  class="chakra-heading css-zr62dw"
                >
                  21st Main Committee
                </h2>
                <p
                  class="chakra-text css-v04ol"
                >
                  March 31, 2021
                </p>
                <p
                  class="chakra-text css-q9k0mw"
                >
                  Introducing the 21st Main Committee Mission of 21st Main Committee School of Computer Science and Engineering Club (CSEC...
                </p>
              </div>
            </div>
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    class="chakra-stack css-ku5d1q"
  >
    <h2
      class="chakra-heading css-df8nxq"
    >
      Let’s work together
    </h2>
    <a
      class="chakra-button css-qklkzv"
      href="./contact"
    >
      <p
        class="chakra-text css-0"
      >
        Contact Us
      </p>
    </a>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</div>
`;
